### PR TITLE
 Document how to enable container domain names. 

### PR DESCRIPTION
**Pull request by foresto**
This is an attempt to help Debian and Ubuntu users discover how to enable domain names for containers, in response to the ubuntu bug report:
https://bugs.launchpad.net/ubuntu/+source/lxc/+bug/1389954

### CODE REVIEW

File: debian/README.Debian [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+Finally, restart the lxc-net service, and the network-manager service if it is
+running on your system.  The commands to do this depend on your init system.
+Here are the common ones:
+
+systemctl restart lxc-net
+systemctl restart network-manager
+
+service lxc-net restart
+service network-manager restart
```
Comment (stgraber) [inline debian/README.Debian]: The Debian way of triggering init scripts is invoke-rc.d which works with all init systems.

Comment (foresto) [inline debian/README.Debian]: Okay, I can rewrite this to use invoke-rc.d for all cases, but Debian 8 (with lxc installed) doesn't seem to have an lxc-net service at all. Any suggestions?

Comment (stgraber) [inline debian/README.Debian]: lxc-net was only upstreamed in LXC 1.1. As Debian 8 ships LXC 1.0.7, it doesn't have the init script, or in fact that feature at all.

Comment (foresto) [inline debian/README.Debian]: Thanks. I just pushed a new commit.

**Comment by foresto** [comment]
Revised to suggest invoke-rc.d for restarting services.

**Comment by stgraber** [comment]
Thanks. One more thing, can you submit this against the daily-ppa branch rather than dpm-vivid?

We typically do changes in that branch and then cherry-pick into the others, also dpm-vivid is likely to stay a bit stale as it's not a LTS release and we're now working on wily (dpm-wily should appear soon).

**Comment by foresto** [comment]
I'll open a new pull request. Either git or github did something surprising to this one whe when I rebased.

**Pull request closed without merging**